ReactiveCocoa/Reusable.swift

```diff
@@ -0,0 +1,10 @@
+import ReactiveSwift
+import enum Result.NoError
+
+public protocol Reusable: class {}
```

Comment:
`prepareForReuse` wasn't listed as a requirement due to `NSCollectionViewItem`. The compiler complains about the availability of `prepareForReuse` only since `OS X application extensions 10.9` even with the presence of `@available` annotation.